user-guide/notebooks/datasets/nldas_compare2_3.ipynb

```diff
@@ -212,24 +212,31 @@
        "  'type': 'Feature',\n",
```

Comment:
<p>This can be simplified to:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">import requests
import folium
import folium.plugins
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/229/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>